From fd390769f5c8801f4e2b74fc9fefb0b6e6ad4b3b Mon Sep 17 00:00:00 2001
From: Dale Hamel <dale.hamel@shopify.com>
Date: Sun, 26 Aug 2018 21:49:11 +0000
Subject: [PATCH 2/5] Fix includes for slotted clang

---
 CMakeLists.txt | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5d2ea28..f3cdc32 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -47,8 +47,9 @@ find_library(libclangSerialization NAMES clangSerialization HINTS ${CLANG_SEARCH
 if(libclangBasic STREQUAL "libclangBasic-NOTFOUND")
   message(FATAL_ERROR "Unable to find clang libraries")
 endif()
-FOREACH(DIR ${LLVM_INCLUDE_DIRS})
-  include_directories("${DIR}/../tools/clang/include")
+
+FOREACH(DIR "/build/lakitu/usr/lib/llvm/6/include/")
+  include_directories("${DIR}")
 ENDFOREACH()
 
 # Set to a string path if system places kernel lib directory in
-- 
2.16.4

