From 7f0d84807288394e93ef23673cab510ac22f9f0a Mon Sep 17 00:00:00 2001
From: Dale Hamel <dale.hamel@shopify.com>
Date: Sun, 26 Aug 2018 22:43:09 +0000
Subject: [PATCH 4/5] trying to allow exceptions

---
 CMakeLists.txt | 11 ++---------
 1 file changed, 2 insertions(+), 9 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 76d5d47..19a23d5 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -80,8 +80,8 @@ set(CMAKE_CXX_STANDARD 11)
 
 endif(NOT PYTHON_ONLY AND ENABLE_CLANG_JIT)
 
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall")
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall ${CXX_ISYSTEM_DIRS}")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -fexceptions")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall ${CXX_ISYSTEM_DIRS} -fexceptions")
 
 add_subdirectory(src)
 add_subdirectory(introspection)
@@ -91,10 +91,3 @@ add_subdirectory(man)
 add_subdirectory(tests)
 add_subdirectory(tools)
 endif(ENABLE_CLANG_JIT)
-
-
-string(REPLACE "-fno-exceptions" " " stripped_cmake_c_flags ${CMAKE_C_FLAGS})
-string(REPLACE "-fno-exceptions" " " stripped_cmake_cxx_flags ${CMAKE_CXX_FLAGS})
-
-set(CMAKE_C_FLAGS "${stripped_cmake_c_flags}")
-set(CMAKE_CXX_FLAGS "${stripped_cmake_cxx_flags} -Wall ${CXX_ISYSTEM_DIRS}")
-- 
2.16.4

